distribution/ddf-common/src/main/resources/security/default.policy

```diff
@@ -30,7 +30,7 @@ deny {
 }
 
 grant
-  codeBase "file:/platform-migration" {
+  codeBase "file:/platform-migration/platform-migratable-api" {
```

Comment:
ℹ️ It was required to add that one because the API defines default methods for interfaces and ends up being on the execution stack even though the implementation extends its privileges